src/calculation/ClauseResultsHelpers.ts

```diff
@@ -214,6 +214,20 @@ export function findAllLocalIdsInStatement(
     } else if (k === 'type' && v === 'Literal' && statement.localId && statement.value === 'false') {
       // If this is a "Literal" expression whose value is false, mark that it `isFalsyLiteral` so we can interpret final results differently
       localIds[statement.localId] = { localId: statement.localId, isFalsyLiteral: true };
+    } else if (k === 'type' && v === 'Not') {
+      // If this is a "Not" expression, we will want to check if it's operand type is 'Equivalent' or 'Equal'
+      // If so, we will want to treat this as a 'Not Equivalent' or 'Not Equal' expression which we can do so
+      // by mapping the 'Equal' or 'Equivalent' clause localId to that of the 'Not' clause
+      if (statement.operand) {
+        if (statement.operand.type === 'Equivalent' || statement.operand.type === 'Equal') {
```

Comment:
This whole set of logic could use a test case. It's one of the few things in here that isn't tested.

Reply:
Good call! I added a unit test for the `!~`...however, I do not think I am able to add a unit test for the `!=` logic at the moment since an update to the translator will be coming soon that puts a localId on the `Equal` expression (but it doesn't right now). Am I correct in this? @birick1 may have additional insight as well